includes/Modules/Site_Verification.php

```diff
@@ -336,4 +375,107 @@ protected function setup_services( Google_Client $client ) {
 			'siteverification' => new Google_Service_SiteVerification( $client ),
 		);
 	}
+
+	/**
+	 * Handles receiving a verification token for a user by the authentication proxy.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function handle_verification_token() {
+		$authentication = $this->authentication;
+		$auth_client    = $authentication->get_oauth_client();
+
+		if (
+			! filter_input( INPUT_GET, 'googlesitekit_verification_token' )
+			|| ! filter_input( INPUT_GET, 'googlesitekit_verification_nonce' )
```

Comment:
Let's assign these to variables so that we don't need to write all this over and over again.